localstack/cli/localstack.py

```diff
@@ -211,10 +217,18 @@ def cmd_logs(follow: bool):
         sys.exit(1)
 
     if follow:
+        num_lines = 0
         for line in DOCKER_CLIENT.stream_container_logs(container_name):
             print(line.decode("utf-8").rstrip("\r\n"))
+            num_lines += 1
+            if tail is not None and num_lines >= tail:
+                break
+
     else:
-        print(DOCKER_CLIENT.get_container_logs(container_name))
+        logs = DOCKER_CLIENT.get_container_logs(container_name)
+        if tail is not None and isinstance(tail, int):
```

Comment:
this `isinstance` check seems unnecessary since click already does the check for us. we also didn't add it to the `if follow` control path so it would be inconsistent also

Reply:
Thank you for your feedback. You are right that the click already checks the type for us, and it is unnecessary to have an additional check `isinstance`. I have removed it in the latest commit.